client/src/components/QandA/QandAContainer.jsx

```diff
@@ -105,29 +105,29 @@ function QandA({ product }) {
   };
 
   return (
-    <div className="qaContainer">
-      Questions & Answers
-      <Search filter={setSearch} />
-      {isLoaded
-        ? <p>Loading...</p>
-        : (
-          <QuestionList
-            product={product}
-            questionHandler={questionHelpfulHandler}
-            questions={currentQuestions}
-            count={questionCount}
-          />
-        )}
-      <div className="buttonContainer">
-        {hasMoreQuestions
-          ? <button type="button" onClick={handleMoreAnsweredQuestions} className="answeredButton">MORE ANSWERED QUESTIONS</button>
-          : null}
-        <button type="button" onClick={() => setShow(true)} className="imageButton"> ADD A QUESTION </button>
-        <AddQuestion
-          addQuestion={addQuestionHandler}
+    <div className="qaContainer container">
+      <div className="qaComponent">
+        QUESTIONS & ANSWERS
+      </div>
+      <div className="qaComponent">
+        <Search filter={setSearch} />
+      </div>
+      <div className="qaComponent">
+        <QuestionList
+          isLoaded={isLoaded}
+          questions={currentQuestions}
           product={product}
-          show={show}
+          questionHandler={questionHelpfulHandler}
+        />
+      </div>
+      <div className="qaComponent">
+        <Buttons
```

Comment:
Good use of creating a button component acting like a container of buttons instead of hard coding all those buttons here